<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfile</class>
 <widget class="QDialog" name="UserProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CREATE NEW USER</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>661</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="2">
     <widget class="QDoubleSpinBox" name="f5_height">
      <property name="maximum">
       <double>200.990000000000009</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="f1_fullname"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Height :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fitness Goal :</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Activity Level :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Age :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gender :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="f2_age"/>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="f3_gender">
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Weight :</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="f6_activity">
      <item>
       <property name="text">
        <string>I am sedentary (little or no exercise)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>I am lightly active (light exercise or sports 1-3 days per week)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>I am moderately active (moderate exercise or sports 3-5 days per week)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>I am very active (hard exercise or sports 6-7 days per week)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>I am super active (very hard exercise or sports and a physical job or 2x training) </string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QDoubleSpinBox" name="f4_weight">
      <property name="maximum">
       <double>150.990000000000009</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fullname :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="f7_goal">
      <item>
       <property name="text">
        <string>Maintain weight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lose weight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gain weight</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
